docs/user/rstudio/ide/guide/code/diagnostics.qmd

```diff
@@ -2,26 +2,23 @@
 title: "Code Diagnostics"
 ---
 
-Copied from <https://support.rstudio.com/hc/en-us/articles/205753617-Code-Diagnostics-in-the-RStudio-IDE>
-
 ## Enabling Diagnostics
 
-Diagnostics can be enabled and options can be set within the `Global Options -> Code -> Diagnostics` editing pane:
+Diagnostics can be enabled and options can be set within the **Global Options** > **Code Diagnostics** editing pane:
 
-\
 ![](images/enable-diagnostics.png){fig-alt="A screenshot of the RStudio Global Options popup, selected on the Code > Diagnostics tab."}
 
 A brief outline of the options available:
 
-**Show Diagnostics for R**
+### Show Diagnostics for R
 
 Toggle the display of R code diagnostics.
 
-**Enable diagnostics within R function calls**
+### Enable diagnostics within R function calls
 
 Controls whether diagnostics are performed within function calls, e.g. `dplyr::select(mtcars, mpg, cyl)`. Toggle this if your code makes heavy use of non-standard evaluation, and RStudio is unable to produce correct diagnostics for you.
 
-**Check arguments to R function calls**
+### Check arguments to R function calls
 
 Try to detect whether a particular call to a function will succeed. The diagnostics engine will report if it detects missing arguments, unmatched arguments, partially matched arguments, and too many arguments.
 
```

Comment:
Missing period in the sentence below:
"Note that it is not necessary for `add_numbers` to exist in the current environment (ie, within the running R session)"

Reply:
Fixed